### DISCUSSION TITLE
Final leaderboard?

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#230] [BaiduKDDCup2022] [answered]
Is it phase 2 the last phase or will it be another one on 15 july? The description has changed.... Thank you

**Comment by fersebas** [comment]
So, there isn't new test data on 15 july?
<img width="1149" alt="image" src="https://user-images.githubusercontent.com/49498032/176884103-c286d02d-2cbe-4154-97dd-f51d8a7cf7ae.png">

**Comment by ramber1836** [comment, marked as answer]
There will be the 3rd phase between July 15 and July 18.

**Reply by fersebas** [reply]
Thanks

**Reply by hongyingyue** [reply]
@ramber1836 

Hey，for my initial understanding, the 3rd phase is still one submit for final score, right?
Or any change currently, like still 3 times for 3 days?

Thanks

**Comment by btrotta** [comment]
@ramber1836 Can you share more details about the 3rd phase? Will it be the same test set as the 2nd phase?
